src/Renderer/ThreeExtended/OBB.js

```diff
@@ -97,4 +98,74 @@ OBB.prototype._cPointsWorld = function _cPointsWorld(points) {
     return points;
 };
 
+// get oriented bounding box of tile
+OBB.extentToOBB = function extentToOBB(extent, minHeight = 0, maxHeight = 0) {
```

Comment:
If you move this function here, you might want to add some asserts at the beginning. At the very least: it'll only work with Geographic coordinates so check `extent`'s crs.